cpp/ps0000/p0000_solution.cpp

```diff
@@ -1,7 +1,8 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main()
+int main() // 1/1
 {
-	cout << "Hello, World!\n";
-  return 0;
-}
\ No newline at end of file
+	cout << "Hello, World!\n"; // 4/4

```

Comment:
- [x] used `<<` for outputting instead of `>>`
- [x] used `\n` or `std::endl` in the right way
- [x] avoided syntax error by typing `;` at the end of the line
- [x] output is as required `Hello, World!` in a single line